src/Symfony/Component/String/CodePointString.php

```diff
@@ -247,7 +247,7 @@ public function split(string $delimiter, int $limit = null, int $flags = null):
         return $chunks;
     }
 
-    public function startsWith($prefix): bool
+    public function startsWith(string|array $prefix): bool
     {
         if ($prefix instanceof AbstractString) {
```

Comment:
This check seems useless, since it's always false ?

Reply:
more probably the argument type should be `string|AbstractString` (not sure why array got mixed in here... array of modifier symbols maybe?)

Reply:
resolved in #41428